src/Agent.Service/Windows/Program.cs

```diff
@@ -8,19 +10,45 @@ static class Program
         /// <summary>
         /// The main entry point for the application.
         /// </summary>
-        static void Main(String[] args)
+        static int Main(String[] args)
         {
             if (args != null && args.Length == 1 && args[0].Equals("init", StringComparison.InvariantCultureIgnoreCase))
-            {                
-                System.Diagnostics.EventLog.WriteEntry(AgentService.EventSourceName, "create event log trace source for vsts-agent service", System.Diagnostics.EventLogEntryType.Information, 100);
-                return;
-            }            
+            {
+                if (!EventLog.Exists("Application"))
+                {
+                    Console.WriteLine("[ERROR] Application event log doesn't exist on current machine.");
```

Comment:
We should add a TODO to loc. I have some changes coming to loc process... will discuss offline.